src/bundle.rs

```diff
@@ -174,13 +176,15 @@ impl<T: Authorization, V> Bundle<T, V> {
         value_balance: V,
         anchor: Anchor,
         authorization: T,
+        assets_burnt: Vec<(AssetId, i64)>,
```

Comment:
Let's keep similar terminology and order:
```rust
/// A bundle of actions to be applied to the ledger.
#[derive(Clone)]
pub struct Bundle<T: Authorization, V> {
    /// The list of actions that make up this bundle.
    actions: NonEmpty<Action<T::SpendAuth>>,
    /// Orchard-specific transaction-level flags for this bundle.
    flags: Flags,
    /// The net value moved out of the Orchard shielded pool.
    ///
    /// This is the sum of Orchard spends minus the sum of Orchard outputs.
    value_balance: V,
    /// Assets intended for burning.
    burn: Vec<(AssetId, V)>,
    /// The root of the Orchard commitment tree that this bundle commits to.
    anchor: Anchor,
    /// The authorization for this bundle.
    authorization: T,
}
```

Reply:
done